helpers/index.ts

```diff
@@ -174,6 +174,9 @@ export const installTemplate = async (
       await createBackendEnvFile(props.root, props);
     }
 
+    // Always create a data directory
+    await makeDir(path.join(props.root, "data"));
```

Comment:
shouldn't this be at the place where we copy the data folder?

Reply:
Thanks, i just moved it to copy data folder:
https://github.com/run-llama/create-llama/pull/279/commits/44e193168bbf41e1a9753d2e8b6dc46cb1034550